pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -69,14 +69,14 @@ import (
 )
 
 const (
-	dockerProcessName     = "dockerd"
-	dockerPidFile         = "/var/run/docker.pid"
-	containerdProcessName = "docker-containerd"
-	containerdPidFile     = "/run/docker/libcontainerd/docker-containerd.pid"
-	maxPidFileLength      = 1 << 10 // 1KB
+	dockerProcessName = "dockerd"
+	dockerPidFile     = "/var/run/docker.pid"
+	containerdPidFile = "/run/docker/libcontainerd/docker-containerd.pid"
+	maxPidFileLength  = 1 << 10 // 1KB
 )
 
 var (
+	containerdProcessNames = []string{"docker-containerd", "containerd"}
```

Comment:
what version the switch happened? Are we still supporting Docker that far back?

Reply:
@SergeyKanzhelev  I think it happened in docker-ce 17.11.
containerd started to run isolatedly after that version.

I have serverl clusters:
- docker 17.03 is using "docker-containerd"
- docker 18.09 & docker 19.03 & 20.10 are using "contained"

Even we don't support dockershim, the kubelet can communicate with containerd directly even we installed docker on the node as well. As I know, kubeadm will not drop all docker supports.

Reply:
makes sense. My sense was that it happened so long ago =). Thank you for confirming!

Reply:
Another easy change on this would be just renaming it to `containerd` as the process name `docker-containerd` is so old.

Reply:
hm, looking at Docker support matrix, it looks like docker 17.03 is [not supported](https://success.mirantis.com/article/maintenance-lifecycle#endoflifeeoldockerengineversions:~:text=End%20of%20Life%20(EOL)%20Docker%20Engine%20Versions). So perhaps yes, simpler fix would be better